### PR TITLE
Update ja-JP.txt

### PR DESCRIPTION
**Pull request by Mitchykins**
If you are applying translations for an issue (or multiple), please list them below (if not, delete this text).

Applying for issue(s):
- #0000
- #0001

### CODE REVIEW

File: data/language/ja-JP.txt [hunk 1]
```diff
@@ -980,11 +980,11 @@ STR_0974    :乗り物
 STR_0975    :軽食店と雑貨屋
 STR_0976    :トイレと案内所
 STR_0977    :新しい輸送ライド
-STR_0978    :New Gentle Rides
-STR_0979    :New Roller Coasters
-STR_0980    :New Thrill Rides
-STR_0981    :New Water Rides
-STR_0982    :New Shops & Stalls
+STR_0978    :新ゲンタレライド
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: 新しい isn't properly conjugated.

File: data/language/ja-JP.txt [hunk 2]
```diff
@@ -993,19 +993,19 @@ STR_0987    :Too many rides/attractions
 STR_0988    :Can't create new ride/attraction...
 STR_0989    :{STRINGID}
 STR_0990    :{SMALLFONT}{BLACK}Construction
-STR_0991    :Station platform
+STR_0991    :駅のプラットフォーム
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: The Japanese word for platform is ホーム.

File: data/language/ja-JP.txt [hunk 3]
```diff
@@ -1063,39 +1063,39 @@ STR_1057    :Ride/attraction name
 STR_1058    :Enter new name for this ride/attraction:
 STR_1059    :Can't rename ride/attraction...
 STR_1060    :Invalid ride/attraction name
-STR_1061    :Normal mode
+STR_1061    :ノーマルモード
 STR_1062    :Continuous circuit mode
 STR_1063    :Reverse-Incline launched shuttle mode
 STR_1064    :Powered launch (passing station)
-STR_1065    :Shuttle mode
+STR_1065    :シャトルモード
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: I think we can go for 往復モード rather than katakana here.

File: data/language/ja-JP.txt [hunk 4]
```diff
@@ -1063,39 +1063,39 @@ STR_1057    :Ride/attraction name
...
-STR_1073    :Race mode
-STR_1074    :Bumper-car mode
-STR_1075    :Swing mode
+STR_1072    :迷路モード
+STR_1073    :レースモード
+STR_1074    :バンパーカーモード
+STR_1075    :スイングモード
 STR_1076    :Shop stall mode
-STR_1077    :Rotation mode
+STR_1077    :回転中モード
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: 回転中 means 'rotating', as in its state. Remove the 中 and it's alright, I think.

File: data/language/ja-JP.txt [hunk 5]
```diff
@@ -1063,39 +1063,39 @@ STR_1057    :Ride/attraction name
...
 STR_1078    :Forward rotation
 STR_1079    :Backward rotation
-STR_1080    :Film: {ENDQUOTES}Avenging aviators{ENDQUOTES}
-STR_1081    :3D film: {ENDQUOTES}Mouse tails{ENDQUOTES}
+STR_1080    :えいが: {ENDQUOTES}Avenging aviators{ENDQUOTES}
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: 映画

File: data/language/ja-JP.txt [hunk 6]
```diff
@@ -1063,39 +1063,39 @@ STR_1057    :Ride/attraction name
...
-STR_1091    :Circus show mode
+STR_1086    :3D えいが: {ENDQUOTES}Storm chasers{ENDQUOTES}
+STR_1087    :3D えいが: {ENDQUOTES}Space raiders{ENDQUOTES}
+STR_1088    :強烈なモード
+STR_1089    :ベルジェルクモード
+STR_1090    :お化け屋敷モード
+STR_1091    :サーカスモード
 STR_1092    :Downward launch
-STR_1093    :Crooked house mode
+STR_1093    :曲がった家モード
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: Crooked house is translated elsewhere as ねじれた家.

File: data/language/ja-JP.txt [hunk 7]
```diff
@@ -1287,34 +1287,34 @@ STR_1281    :Structures
...
-STR_1290    :{COMMA16} ships
-STR_1291    :Ship {COMMA16}
-STR_1292    :cabin
-STR_1293    :cabins
-STR_1294    :Cabin
-STR_1295    :Cabins
-STR_1296    :{COMMA16} cabin
-STR_1297    :{COMMA16} cabins
-STR_1298    :Cabin {COMMA16}
+STR_1285    :シップ
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: I think シップ is fine for katakana words like in 'pirate ship', but here I'd like to use 船 (ふね)

File: data/language/ja-JP.txt [hunk 8]
```diff
@@ -2067,9 +2067,9 @@ STR_2058    :クーポン
 STR_2059    :ポップコーン
 STR_2060    :ホットドッグ
 STR_2061    :a Tentacle
-STR_2062    :a Hat
+STR_2062    :帽子
 STR_2063    :a Toffee Apple
-STR_2064    :a T-Shirt
+STR_2064    :Tシャツ
 STR_2065    :a Doughnut
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: These strings are translated elsewhere already — feel free to pick them up, too.

File: data/language/ja-JP.txt [hunk 9]
```diff
@@ -993,19 +993,19 @@ STR_0987    :Too many rides/attractions
 STR_0988    :Can't create new ride/attraction...
 STR_0989    :{STRINGID}
 STR_0990    :{SMALLFONT}{BLACK}Construction
-STR_0991    :Station platform
+STR_0991    :駅のプラットフォーム
 STR_0992    :{SMALLFONT}{BLACK}Demolish entire ride/attraction
-STR_0993    :Demolish ride/attraction
-STR_0994    :Demolish
+STR_0993    :ライド/アトラクション解体ます
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: 解体 is a noun; you can make it a verb by using する or します. Also make it take an object: をします。

File: data/language/ja-JP.txt (hunk 2 above)
Comment (AaronVanGeffen) [reply data/language/ja-JP.txt]: I checked how RCT3 does this. Turns out they do away with station terminology altogether and use 乗り場 instead. Seems like a fine translation to me.

**Comment by Gymnasiast** [comment]
The improvements in this PR have been merged via #1261 (crediting @Mitchykins ).

**Pull request closed without merging**